packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -2388,11 +2490,1768 @@ void main() {
     );
     // Indicator with Stack widget
     final RenderBox firstIndicator = tester.renderObject(find.byType(Icon).first);
-    expect(firstIndicator.localToGlobal(Offset.zero).dx, 24.0);
+    expect(firstIndicator.localToGlobal(Offset.zero).dx, 28.0);
     // Indicator without Stack widget
     final RenderBox lastIndicator = tester.renderObject(find.byType(Icon).last);
-    expect(lastIndicator.localToGlobal(Offset.zero).dx, 24.0);
+    expect(lastIndicator.localToGlobal(Offset.zero).dx, 28.0);
   });
+
+  group('Material 2', () {
```

Comment:
what a test! 🤯

Reply:
Yeah this was probably hard to read, but it was just a copy of all the original M2 tests that failed with M3. Then I fixed the original tests to work with M3. This way we still have both, but we can easily remove this M2 group when the migration is complete.